zhaquirks/xiaomi/aqara/switch_acn047.py

```diff
@@ -0,0 +1,173 @@
+"""Aqara T2 relay device."""
+from zigpy import types as t
+from zigpy.profiles import zha
+from zigpy.zcl.clusters.general import (
+    AnalogInput,
+    Basic,
+    DeviceTemperature,
+    Groups,
+    Identify,
+    MultistateInput,
+    OnOff,
+    Ota,
+    Scenes,
+    Time,
+)
+from zigpy.zcl.clusters.homeautomation import ElectricalMeasurement
+from zigpy.zcl.clusters.smartenergy import Metering
+
+from zhaquirks.const import (
+    DEVICE_TYPE,
+    ENDPOINTS,
+    INPUT_CLUSTERS,
+    MODELS_INFO,
+    OUTPUT_CLUSTERS,
+    PROFILE_ID,
+)
+from zhaquirks.xiaomi import (
+    AnalogInputCluster,
+    BasicCluster,
+    ElectricalMeasurementCluster,
+    MeteringCluster,
+    XiaomiAqaraE1Cluster,
+    XiaomiCustomDevice,
+)
+
+
+class OppleCluster(XiaomiAqaraE1Cluster):
+    """Opple cluster."""
+
+    class SwitchType(t.enum8):
+        """Switch type."""
+
+        Toggle = 0x01
+        Momentary = 0x02
+        NoSwitch = 0x03
+
+    class StartupOnOff(t.enum8):
+        """Startup mode."""
+
+        On = 0x00
+        Previous = 0x01
+        Off = 0x02
+        Toggle = 0x03
+
+    class DecoupledMode(t.enum8):
+        """Decoupled mode."""
+
+        Decoupled = 0x00
+        ControlRelay = 0x01
+
+    class Mode(t.enum8):
+        """Mode."""
+
+        Power = 0x00
+        Pulse = 0x01
+        Dry = 0x03
+
+    attributes = {
+        0x000A: ("switch_type", SwitchType, True),
+        0x0517: ("startup_on_off", StartupOnOff, True),
+        0x0200: ("decoupled_mode", DecoupledMode, True),
+        0x02D0: ("interlock", t.Bool, True),
+        0x0289: ("mode", Mode, True),
```

Comment:
Should we rename this `mode` attribute slightly, so it's not confused with the `0x0009`/`mode` attribute that's used on most other Aqara devices?

Maybe `switch_mode`?